crates/utils/translations/en.json

```diff
@@ -0,0 +1,18 @@
+{
+  "registration_approved_subject": "Registration approved for {username}",
+  "registration_approved_body": "Your registration application has been approved. Welcome to {hostname}!",
+  "password_reset_subject": "Password reset for {username}",
+  "password_reset_body": "<h1>Password Reset Request for {username}</h1><br><a href={reset_link}>Click here to reset your password</a>",
+  "verify_email_subject": "Verify your email address for {hostname}",
+  "verify_email_body": "Please click the link below to verify your email address for the account @{username}@{hostname}. Ignore this email if the account isn't yours.<br><br>, <a href=\"{verify_link}\">Verify your email</a>",
+  "email_verified_subject": "Email verified for {username}",
+  "email_verified_body": "Your email has been verified.",
+  "notification_post_reply_subject": "Reply from {username}",
+  "notification_post_reply_body": "<h1>Post reply</h1><br><div>{username} - {comment_text}</div><br><a href={inbox_link}>inbox</a>",
+  "notification_comment_reply_subject": "Reply from {username}",
+  "notification_comment_reply_body": "<h1>Comment reply</h1><br><div>{username} - {comment_text}</div><br><a href={inbox_link}>inbox</a>",
+  "notification_mentioned_by_subject": "Mentioned by {username}",
+  "notification_mentioned_by_body": "<h1>Person Mention</h1><br><div>{username} - {comment_text}</div><br><a href={inbox_link}>inbox</a>",
+  "notification_private_message_subject": "Private message from {username}",
+  "notification_private_message_body": "<h1>Private message</h1><br><div>{username} - {message_text}</div><br><a href={inbox_link}>inbox</a>"
+}
```

Comment:
Nice, this is pretty easy, as long as its positional then.

Not super looking forward to setting up a third weblate for lemmy... but its pry gonna need one.

Reply:
Yes, at least temporarily until the rosetta crate is updated.